include/seahorn/OperationalSemantics.hh

```diff
@@ -213,6 +225,15 @@ public:
     m_filter.insert(begin, end);
   }
 
+  /// Crab stuff
+  /// \brief Creates a crab's cfg builder manager
+  void mkCrabAnalysis(const llvm::Module &M, seadsa::GlobalAnalysis &dsa,
```

Comment:
I will stop commenting on each individual line. Nothing crab related should be anywhere except for where it is used in opsem2